src/System.IO.FileSystem/tests/Path/GetPathRoot.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public class Path_GetPathRoot : FileSystemTest
+    {
+        [Fact]
+        public void NullAsPath_ReturnsNull()
+        {
+            Assert.Null(Path.GetPathRoot(null));
```

Comment:
We already have a test for this: https://github.com/dotnet/corefx/blob/master/src/System.Runtime.Extensions/tests/System/IO/PathTests.cs#L177